src/ecalc/libraries/libecalc/common/libecalc/output/results/configs/configs.py

```diff
@@ -597,6 +609,7 @@ def aggregator(frequency: Frequency) -> Aggregator:
                     title="Total Power Consumed By Gas-Turbine Driven Compressors",
                     unit=Unit.GIGA_WATT_HOURS,
                     query=FuelConsumerPowerConsumptionQuery(
+                        # installation_category="FIXED",
```

Comment:
please add a comment why this is commented out

Reply:
Ah, forgot to remove - part of debugging in code development process. Thanks.

Reply:
It is fixed now